integration/integration_test.go

```diff
@@ -4855,6 +4855,53 @@ func (s *IntSuite) TestBPFSessionDifferentiation(c *check.C) {
 	c.Fatalf("Failed to find command events from two different sessions.")
 }
 
+// TestExecEvents tests if exec events were emitted with and without PTY allocated
+func (s *IntSuite) TestExecEvents(c *check.C) {
+	s.setUpTest(c)
+	defer s.tearDownTest(c)
+	tr := utils.NewTracer(utils.ThisFunction()).Start()
+	defer tr.Stop()
+
+	lsPath, err := exec.LookPath("ls")
+	c.Assert(err, check.IsNil)
+
+	// Creates new teleport cluster
+	main := s.newTeleport(c, nil, true)
+	defer main.StopAll()
+
+	var execTests = []struct {
+		isInteractive bool
+		outCommand    string
+	}{
+		{
+			isInteractive: true,
```

Comment:
Can you add a comment or description field that explains what each test case is about?
It could be used as following:
```go
	var execTests = []struct {
		comment       string
		isInteractive bool
		outCommand    string
	}{
		// ...
	}
	for _, tt := range execTests {
		// Create a comment value
		comment := check.Commentf(tt.comment)
		// Each time an assertion fails, it will print the comment so it's clear which test case it was:
		c.Assert(err, check.IsNil, comment)
	}
```